event_sourcery/interfaces/outbox_storage_strategy.py

```diff
@@ -1,10 +1,11 @@
 import abc
 from typing import Iterator, Tuple
+from uuid import UUID
 
 from event_sourcery.dto import RawEvent
 from event_sourcery.types import StreamId
 
-EntryId = int
+EntryId = int | UUID
```

Comment:
Why like this...? Unions are pesky later

Reply:
Yes. I agree with you. I have removed this change.
But, IMO we need to rethink this API. Let's assume that ESDB int is not a valid entry id. Now it's int but we need a session to discuss outbox.